packages/lib/src/utils/Formatters/formatter.test.ts

```diff
@@ -0,0 +1,18 @@
+import { digitsOnlyFormatter } from "./formatters";
+
+function getAllCharactersArray() { 
+    const allCharacters:Array<string> = [];
+    for (let i=32; i<127; i++) { allCharacters.push(String.fromCharCode(i)); }
```

Comment:
Does the generated chars array include symbols as well? So we can make sure that it indeed the regex will remove everything except digits

Reply:
Ah ok, I thought it generated only the alphabet letters.